pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -411,6 +411,10 @@ type TiKVSpec struct {
 	// Config is the Configuration of tikv-servers
 	// +optional
 	Config *TiKVConfig `json:"config,omitempty"`
+
+	// RecoverFailover indicates that Operator can recover the failover Pods
+	// +optional
+	RecoverFailover bool `json:"recoverFailover,omitempty"`
```

Comment:
AutoRecover?

Reply:
I think no need to emphasize `auto` here, all of the configurations in the `spec` will be done automatically by the Operator.

Reply:
How about `allowRecoverFailover`? It same hard to understand what this variable will affect through its name.